package.json

```diff
@@ -1,123 +1,17 @@
-    "node-test": "mocha ./test/{integration,unit}/**/*-test.js",
-    "test": "npm run-script lint:js && npm run-script node-test"
-  },
-  "dependencies": {
-    "babel-plugin-istanbul": "^6.0.0",
-    "body-parser": "^1.19.0",
-    "ember-cli-version-checker": "^5.1.1",
-    "fs-extra": "^9.0.0",
-    "istanbul-lib-coverage": "^3.0.0",
-    "istanbul-lib-report": "^3.0.0",
-    "istanbul-reports": "^3.0.2",
-    "node-dir": "^0.1.17",
-    "walk-sync": "^2.1.0"
+    "test": "mocha test-packages/*-test.js"
   },
   "devDependencies": {
-    "@ember/optional-features": "^1.3.0",
-    "@glimmer/component": "^1.0.0",
-    "@glimmer/tracking": "^1.0.0",
-    "babel-eslint": "^10.1.0",
-    "broccoli-asset-rev": "^3.0.0",
     "chai": "^4.2.0",
     "chai-files": "^1.4.0",
-    "ember-auto-import": "^1.5.3",
-    "ember-cli": "~3.16.2",
-    "ember-cli-addon-tests": "^0.11.0",
-    "ember-cli-babel": "^7.17.2",
-    "ember-cli-dependency-checker": "^3.2.0",
-    "ember-cli-htmlbars": "^4.2.2",
-    "ember-cli-inject-live-reload": "^2.0.2",
-    "ember-cli-sri": "^2.1.1",
-    "ember-cli-uglify": "^3.0.0",
-    "ember-disable-prototype-extensions": "^1.1.3",
-    "ember-exam": "1.0.0",
-    "ember-export-application-global": "^2.0.1",
-    "ember-load-initializers": "^2.1.1",
-    "ember-maybe-import-regenerator": "^0.1.6",
-    "ember-qunit": "^4.6.0",
-    "ember-resolver": "^7.0.0",
-    "ember-source": "~3.16.0",
-    "ember-source-channel-url": "^2.0.1",
-    "ember-template-lint": "^2.8.0",
-    "ember-try": "^1.4.0",
-    "eslint": "^7.1.0",
-    "eslint-config-prettier": "^6.11.0",
-    "eslint-plugin-ember": "^8.5.2",
-    "eslint-plugin-node": "^11.1.0",
-    "eslint-plugin-prettier": "^3.1.4",
-    "fixturify": "^0.3.4",
-    "glob": "^7.1.3",
-    "loader.js": "^4.7.0",
+    "execa": "^4.0.3",
     "mocha": "^5.2.0",
-    "prettier": "^2.0.5",
-    "qunit-dom": "^1.0.0",
-    "release-it": "^13.6.0",
-    "release-it-lerna-changelog": "^2.3.0",
-    "rimraf": "^2.6.2",
-    "sinon": "^4.2.2"
-  },
-  "engines": {
-    "node": "10.* || >= 12"
-  },
-  "publishConfig": {
-    "registry": "https://registry.npmjs.org"
-  },
-  "changelog": {
-    "repo": "kategengler/ember-cli-code-coverage",
-    "labels": {
-      "breaking": ":boom: Breaking Change",
-      "enhancement": ":rocket: Enhancement",
-      "bug": ":bug: Bug Fix",
-      "documentation": ":memo: Documentation",
-      "internal": ":house: Internal"
-    }
-  },
-  "ember": {
-    "edition": "octane"
-  },
-  "ember-addon": {
-    "configPath": "tests/dummy/config",
-    "before": "ember-cli-typescript"
-  },
-  "release-it": {
-    "plugins": {
-      "release-it-lerna-changelog": {
-        "infile": "CHANGELOG.md",
-        "launchEditor": true
-      }
-    },
-    "git": {
-      "tagName": "v${version}"
-    },
-    "github": {
-      "release": true,
-      "tokenRef": "GITHUB_AUTH"
-    },
-    "npm": {
-      "publish": false
-    }
+    "rimraf": "^2.6.2"
   }
```

Comment:
Should bring this back, with nearly the same configuration as https://github.com/ember-fastboot/ember-cli-fastboot/pull/783